Tasks/KubernetesManifestV0/src/utils/CanaryDeploymentHelper.ts

```diff
@@ -32,17 +33,7 @@ export function deleteCanaryDeployment(kubectl: Kubectl, manifestFilePaths: stri
 
     // create delete cmd prefix
     let argsPrefix: string;
-    argsPrefix = createCanaryObjectsArgumentString(inputManifestFiles, includeServices);
-
-    // append delete cmd args as suffix (if present)
-    const args = utils.getDeleteCmdArgs(argsPrefix, TaskInputParameters.args);
-    tl.debug('Delete cmd args : ' + args);
-
-    if (!!args && args.length > 0) {
-        // run kubectl delete cmd
-        const result = kubectl.delete(args);
-        utils.checkForErrors([result]);
-    }
+    createCanaryObjectsArgumentString(kubectl, inputManifestFiles, includeServices);
```

Comment:
Rename the function and remove the declaration of the above variable

Reply:
Okay